docs/src/environments/_index.md

```diff
@@ -54,106 +89,91 @@ Each new environment you create is considered a **child** of the **parent** envi
-the remaining features [sync](../other/glossary.md#sync) with the Sprint environment.
-This ensures their working environment is up-to-date with the latest code.
+Example of an Agile workflow :
 
-![Features from the Sprint environment are synced with the Feature 2 environment](/images/workflow/sync.svg "0.2")
+1.  The admin [branches](../other/glossary.md#branch) the Live environment to create a Sprint environment.
 
-When the sprint is complete and all features merged into the Sprint environment,
-the administrator can make a backup of the live site.
-Then they can merge the Sprint environment into the Live environment.
+2.  The admin gives each developer permission to branch the Sprint environment to create new feature environments.
 
-![Features from the Sprint environment are merged into the Live environment](/images/workflow/merge-live.svg "0.2")
+    ![Agile branches with Live as the top parent, Sprint as a child, and Feature 1 and Feature 2 as children of Sprint](/images/workflow/branches.svg "0.2")
 
-The administrator can then sync the next sprint's environment with the Live environment
-to repeat and continue the development process.
+3.  Feature 1 is developed and work is reviewed by accessing the deployed Feature 1 environment.
 
-## Naming conventions
+4.  When the review is done, Feature 1 is [merged](../other/glossary.md#merge) into the Sprint environment.
+
+5.  The remaining features [sync](../other/glossary.md#sync) with the Sprint environment.
+    This ensures their working environment is up-to-date with the latest code.
+
+    ![Features from the Sprint environment are synced with the Feature 2 environment](/images/workflow/sync.svg "0.2")
+
+6.  When the sprint is complete and all features merged into the Sprint environment,
+    the admin makes a backup of the live site.
+
+7.  The admin merges the Sprint environment into the Live environment.
+
+    ![Features from the Sprint environment are merged into the Live environment](/images/workflow/merge-live.svg "0.2")
+
+8.  The admin syncs the next sprint's environment with the Live environment
+    to repeat and continue the development process.
+
+### Naming conventions
 
 You can organize and work with your development environments in many different ways.
 It can help to introduce a convention for how you name and structure your environments.
 
-The name should represent the environment's purpose.
-Is it a Staging site to show to your client? Is it an implementation of a new feature?
-Is it a hotfix?
-
-If you use Agile, for example, you could create hierarchical environments and name them like this:
+For each environment, choose a name that represents what the environment is for.
+If you use Agile, for example, you can adopt a naming convention similar to the following:
 
 ```text
 Live
-  Sprint1
-    Feature1
-    Feature2
-    Feature3
-  Sprint2
-    Feature1
-    Feature2
+ Sprint1
+   Feature1
+   Feature2
+   Feature3
+ Sprint2
+   Feature1
+   Feature2
```

Comment:
:clipboard: A single space at the start of a line isn't great for being able to see the difference. I'd always have at least 2 for indentation
```suggestion
  Sprint1
    Feature1
    Feature2
    Feature3
  Sprint2
    Feature1
    Feature2
```